scripts/build-image-runtime

```diff
@@ -8,6 +8,7 @@ source ./scripts/version.sh
 DOCKER_BUILDKIT=${DOCKER_BUILDKIT:-1} docker image build \
     --build-arg TAG=${VERSION} \
     --build-arg KUBERNETES_VERSION=${KUBERNETES_VERSION} \
+    --build-arg CACHEBUST=$(date +%s%N) \
```

Comment:
CACHEBUST should default to the value of $TAG unless specified in the env. this can be done in the version.sh script

Reply:
this will allow for caching by default but guarantee a cache miss when building tagged releases (except for re-tags, which we shouldnt be doing anyway). this way you have the option of overridding CACHEBUST in the drone CI (or locally) to force it

Reply:
As per offline discussion, we will use the `date` approach for now, and eventually move to a more standardized method that allows for selective cache invalidation.